packages/numpy/CONFIG

```diff
@@ -15,6 +16,9 @@ PATCHES_1.9.2 = \
 PATCHES_1.16.4 = \
     avoid-writing-in-sources.patch
 
+#PATCHES_1.17.5 = \
+#	avoid-writing-in-sources.patch
```

Comment:
Please either include the patch or not: make up your mind!

Reply:
The patch prevents the build writing some version file into the sources directory, but the build fails if the file isn't present.
Will remove the patch for now and consider a better fix along with the rest of issue #9.

Reply:
Ok.  Interesting that it works for 1.16.4 but not 1.17.5; guess we leave it for 1.16.

Thanks very much for chasing up all these little build annoyances!